internal/security/bootstrapper/command/setupacl/command.go

```diff
@@ -184,24 +177,6 @@ func (c *cmd) GetCommandName() string {
 	return CommandName
 }
 
-func (c *cmd) getSecretStoreClient() (secrets.SecretStoreClient, error) {
-	clientConfig := types.SecretConfig{
-		Type:     c.configuration.SecretStore.Type,
-		Protocol: c.configuration.SecretStore.Protocol,
-		Host:     c.configuration.SecretStore.Host,
-		Port:     c.configuration.SecretStore.Port,
-	}
-
-	secretstoreClient, err := secrets.NewSecretStoreClient(clientConfig, c.loggingClient, c.client)
```

Comment:
With this removed, the GenerateConsulToken can now be removed from SecretStoreClient interface in go-mod-secrets, correct. And its `token` parameter can be removed and implementation updated , i.e. now only uses the service vault token.

Reply:
Right. Good point, i'll created an issue for that tech-debt.

Reply:
issue https://github.com/edgexfoundry/go-mod-secrets/issues/104 created